simulator/tabletop.py

```diff
@@ -79,3 +79,24 @@ def start_app_on_all(self, app_type):
   def run(self):
     """ Runs the tabletop simulation indefinitely. """
     self.__canvas.wait_for_events()
+  
+  def draw_grid(self):
```

Comment:
Function should have a docstring explaining what it does and what arguments it takes.